test/e2e/elasticache/replication_group/coverage_tests/creation.sh

```diff
@@ -0,0 +1,211 @@
+#!/usr/bin/env bash
```

Comment:
@echen-98 would you mind removing the `coverage_tests` directory and just placing all these tests in `test/e2e/elasticache/replication_group/` please? Thanks!

Reply:
Done!